### ISSUE TITLE
FutureWarning in rule snakemake -j1 results/_resources/plots/scal_conv_pp-capacity_net_el.png

### ISSUE THREAD

**Issue by Stefanie08** [rl-institut/oemof-B3#347]
Executing the rules 
```
snakemake -j1 results/_resources/plots/scal_conv_pp-capacity_net_el.png
```
or 
```
snakemake -j1 plot_all_resources
```
raises a FutureWarning concerning pandas. The pandas version used for this warning message is 3.0.2. The error message states teh following:
```
~\oemof-B3\oemof_b3\tools\data_processing.py:487: FutureWarning: The provided callable <built-in function sum> is currently using SeriesGroupBy.sum. In a future version of pandas, the provided callable will be used directly. To keep current behavior pass 'sum' instead. 
return df.groupby(groupby, sort=False, dropna=False).agg(agg_method)
```

This should be adapted to avoid any future errors.

**Comment by MaGering** [comment]
Thanks for finding this! Let's try to fix this within [Milestone v0.0.3](https://github.com/rl-institut/oemof-B3/milestone/3).

**Comment by MaGering** [comment]
Running `snakemake -j1 results/_resources/plots/scal_conv_pp-capacity_net_el.png` and `snakemake -j1 plot_all_resources` locally on branch [features/update_tabular](https://github.com/rl-institut/oemof-B3/tree/features/update_tabular) I could not reproduce the FutureWarning. Could you please check again if you get it @Stefanie08 and otherwise close this issue?

I'll change the Milestone to a later release date.

Thank You!
